bundle/manifests/everest.percona.com_databaseclusterrestores.yaml

```diff
@@ -158,86 +102,11 @@ spec:
               completed:
                 format: date-time
                 type: string
-              conditions:
-                items:
-                  description: "Condition contains details for one aspect of the current
-                    state of this API Resource. --- This struct is intended for direct
-                    use as an array at the field path .status.conditions.  For example,
-                    \n type FooStatus struct{ // Represents the observations of a
-                    foo's current state. // Known .status.conditions.type are: \"Available\",
-                    \"Progressing\", and \"Degraded\" // +patchMergeKey=type // +patchStrategy=merge
-                    // +listType=map // +listMapKey=type Conditions []metav1.Condition
-                    `json:\"conditions,omitempty\" patchStrategy:\"merge\" patchMergeKey:\"type\"
-                    protobuf:\"bytes,1,rep,name=conditions\"` \n // other fields }"
```

Comment:
so it was like a "nice to have" which wasn't implemented anyway, so we deleted it?

Reply:
Actually we didn't even set this anywhere. I don't know what was the initial intention for adding it to the API but it wasn't used anywhere.